user_guides/saaj/src/main/jbake/content/attributes.conf

```diff
@@ -18,7 +18,7 @@
 //
 // The environment variable used to specify the home directory
 // for the technology.  Used in config.inc.
-:TechnologyHomeEnv: WOMBAT_HOME
+:TechnologyHomeEnv: TS_HOME
```

Comment:
can you fully fix this file, please?

`TechnologyRI` is `Jakarta SOAP with Attachments Implementation 2.0.0`
`TechnologyRIURL` is either `https://eclipse-ee4j.github.io/metro-saaj/` or `https://projects.eclipse.org/projects/ee4j.metro` (former is a subset of the latter and both are included in GF)
`SpecificationInquiryList` is `jaxws-dev@eclipse.org`